src/Form/RdfForm.php

```diff
@@ -42,10 +42,12 @@ public function save(array $form, FormStateInterface $form_state) {
     /** @var \Drupal\rdf_entity\Entity\Rdf $entity */
     $entity = $this->getEntity();
     $entity->save();
-    $form_state->setRedirect(
-      'entity.rdf_entity.canonical',
-      ['rdf_entity' => $entity->id()]
-    );
+    if (!$form_state->getRedirect()) {
```

Comment:
That was also one of my local changes. I will accept it because I also find it a good solution, though, please note, that Nodes are not doing it like this.

Reply:
Well... node is wrong :)